### DISCUSSION TITLE
Gradient coloring of a custom overlay that consists of numerical scale

### DISCUSSION THREAD

**Discussion by vizsim** [Helium314/SCEE#642] [General]
I created an overlay to identify the width of non-separated cycleways. It works pretty well, but I was wondering if it is possible to color the ways with known widths according to their corresponding width on a gradient colorscale f.e. vidiris instead of the random colors? That can be helpful to identify inconsistencies and would be a nice feature anyways (also for maxspeed).

Also is there a way to show the way direction in SCEE, so right or left become clear?

### Filtering details

```
highway and (cycleway~lane|track or cycleway:right~lane|track or
cycleway:left ~lane|track or cycleway:both~lane|track)
```

### Coloring details
`.*:width*`

![Screenshot_20240908-103142](https://github.com/user-attachments/assets/d6e1e77d-dc94-414f-ab1d-332ee501667b)

**Comment by Helium314** [comment]
> if it is possible to color the ways with known widths according to their corresponding width on a gradient colorscal

Currently not. Implementing this would require quite a bit of work, as the range needs to be definen manually, and values that are not purely numeric should also work.

> Also is there a way to show the way direction in SCEE, so right or left become clear?

There is some setting, according to readme it should be in display settings

**Reply by vizsim** [reply]
> There is some setting, according to readme it should be in display settings

Thanks for the hint! It works nicely.

> Currently not. Implementing this would require quite a bit of work, as the range needs to be definen manually, and values that are not purely numeric should also work.

I see. But thank you anyway.
